BUILD/phylums/banish.dat

```diff
@@ -0,0 +1,13 @@
+dude	loc:The Black Forest
+dude	loc:Twin Peak
+dude	loc:The Red Zeppelin;item:Glark Cable>2
+dude	loc:Whitey's Grove;quest:questL11Palindome<=3
+beast	loc:The Hidden Park
+beast	loc:Inside the Palindome;quest:questL11Palindome<=3
+beast	loc:The Penultimate Fantasy Airship
+undead	loc:The Haunted Library	item:Killing Jar>0
+undead	loc:The Haunted Wine Cellar
+undead	loc:The Unquiet Garves
+undead	loc:The Haunted Boiler Room
+undead	loc:The Middle Chamber
+construct	loc:Noob Cave
```

Comment:
this also triggers on the "fake" combats that we pretend to do in the Noob Cave, like the 151 infantryman

Not sure what to do about this. Could just remove it entirely as I don't think we ever go to the Noob Cave seriously, and the "clear phylum banish" can special-case it

While you currently go to the Noob Cave to clear a phylum banish by banishing constructs I feel like there are better options, if unlocked:
* Cobb's Knob Harem goblins, and we only want to face goblins in pure goblin zones, like this one. Also advances the superlikely.
* Outskirts is okay at a push but has some unskippable NCs, so is dangerous
* Back Alley clears bugs or hobos but also has unskippable NCs

because we do want constructs in the Wine Cellar / Laundry Room.